### PR TITLE
Added theses from Licha and Ostheimer

### PR DESCRIPTION
**Pull request by SundermannC**

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+@mastersthesis{Licha23,
+	author = {Licha, Lars},
+	title = {{Cutting Edge T-Wise Sampling with ddnnife}},
+	school = UniUlm,
+	address = {Germany},
+	type = {Bachelor's Thesis},
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting},
+	month = JAN,
+	year = 2023
 }
```
Comment (tthuem) [inline literature.bib:10]: note = {To appear}

missing in case a publication is planned

File: literature.bib [hunk 2]
```diff
@@ -105,6 +71,18 @@ @mastersthesis{Tschada21
...
 
+@article{SHN+:EMSE23,
+	author = {Chico Sundermann and Tobias He\ss{} and Michael Nieke and Paul Maximilian Bittner and Jeffrey M. Young and Thomas Th\"um and Ina Schaefer},
+	title = {{Evaluating State-of-the-Art \#SAT Solvers on Industrial Configuration Spaces}},
+	journal = EMSE,
+	publisher = Springer,
+	year = 2023,
+	month = JAN,
+	volume = 28,
+	doi = {https://doi.org/10.1007/s10664-022-10265-9},
```
Comment (tthuem) [inline literature.bib:82]: doi without https...

see other entries

File: literature.bib [hunk 3]
```diff
@@ -119,6 +97,42 @@ @proceedings{VaMoS23
...
+@mastersthesis{Ostheimer22,
+	author = {Ostheimer, Lukas},
+	title = {{Identification of Variance Driving Features in Feature Models}},
+	school = UniUlm,
+	address = {Germany},
+	type = {Bachelor's Thesis},
+	typo3Tags = {SupervisorCS, SupervisorTT},
+	month = DEC,
+	year = 2022
+}
```
Comment (tthuem) [inline literature.bib:109]: To appear?

Not yet published

File: literature.bib [hunk 4]
```diff
@@ -297,6 +311,17 @@ @inproceedings{SBE+:EASE22
...
+@mastersthesis{Schiessl22,
+	author = {Schiessl, Dominik},
+	title = {{An Incremental \#SAT Solver for Efficient Analysis of Feature Models}},
+	school = UniUlm,
+	address = {Germany},
+	type = {Master's Thesis},
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting},
+	month = JAN,
+	year = 2022
+}
```
Comment (tthuem) [inline literature.bib:323]: To appear?

File: literature.bib (hunk 3 above)
Comment (SundermannC) [reply literature.bib:109]: Currently not planned to be published.

File: literature.bib (hunk 4 above)
Comment (pmbittner) [reply literature.bib:405]: Publications that are flagged as "to appear" should be at the top of the file.

**Review by pmbittner — CHANGES_REQUESTED** [review]

**Review by pmbittner — APPROVED** [review]

**Pull request merged**